tests/src/Common/Coreclr.TestWrapper/CoreclrTestWrapperLib.cs

```diff
@@ -71,23 +71,27 @@ public int RunTest(string executable, string outputFile, string errorFile)
 
                 process.Start();
 
-                Task copyOutput = process.StandardOutput.BaseStream.CopyToAsync(outputStream);
-                Task copyError = process.StandardError.BaseStream.CopyToAsync(errorStream);
+                var cts = new CancellationTokenSource();
+                Task copyOutput = process.StandardOutput.BaseStream.CopyToAsync(outputStream, 4096, cts.Token);
```

Comment:
MSDN says the default size is 81920.  Assuming the current default isn't 4096, any reason not to use the default?